aggregate/example/run.sh

```diff
@@ -0,0 +1,2 @@
+module load python/3.10/modulefile
+./../aggregate -m /data/user/shared_projects/mavisp/RAD51B/interactome/mentha2pdb/O15315.csv -s /data/user/elenikia/PPI2PDB/string2pdb/example/O15315_string_interactors.csv
```

Comment:
our example should be self-standing, which means all inputs and outputs should be in the main folder (unless they are e.g. a large dataset/database or similar), so these files should be added in the examples and not included as absolute paths